### DISCUSSION TITLE
Asking for help regarding new project

### DISCUSSION THREAD

**Discussion by Adarsh-jaiss** [TechStudent10/techstudent10#34] [Q&A]
Hey, I am adarsh ☺
I am currently working on an open source project with @KendallDoesCoding . 
When I mentioned that we want someone who can code JavaScript . So he, recommended you 😃. 

So I would love to ask you, will you help us in that project 😃

Here's the link of the project :https://github.com/Adarsh-jaiss/journey-book

**Comment by TechStudent10** [comment]
I could help, but with what? What kind of functionality do you want to add?

**Reply by KendallDoesCoding** [reply]
https://github.com/Adarsh-jaiss/journey-book/issues/1
